### PR TITLE
chore: move OrderSummary to FastStore UI

### PR DESCRIPTION
**Pull request by icazevedo**
## What's the purpose of this pull request?

Moves `OrderSummary` to FastStore UI

## How does it work?

Uses new `OrderSummary` component from FastStore UI.

## How to test it?

There should be no visual differences between this version of the component when compared with the one in https://vtexfaststore.com. You can see it after adding an item to the minicart.

Screen Shot 2022-09-08 at 10 59 28

## Checklist

<em>You may erase this after checking them all :wink:</em>

**PR Title and Commit Messages**
- [x] PR title and commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
  - Available prefixes: `feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, and `test`

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Features`, `Bug fixes`, `Chore`, `Documentation`, `Style changes`, `Refactoring`, `Performance`, and `Test`
- [x] Added the component, hook, or path name in-between backticks (\`\`) - *if applicable, e.g., `ComponentName` component, `useWindowDimensions` hook*

**Dependencies**
- [x] Committed the `yarn.lock` and `bun.lockb` file when there were changes to the packages

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*
- [x] For documentation changes, ping `@carolinamenezes` or `@PedroAntunesCosta` to review and update

### CODE REVIEW

File: src/components/cart/OrderSummary/OrderSummary.tsx [hunk 1]
```diff
@@ -20,23 +22,17 @@ function OrderSummary({
...
-      <li className="text__title-subsection">
-        <span>Total</span>
-        <span>{useFormattedPrice(total)}</span>
-      </li>
+    <div className={styles.fsOrderSummary}>
+      <UIOrderSummary
+        subtotalLabel={`Subtotal (${numberOfItems} products)`}
+        subtotalValue={useFormattedPrice(subTotal)}
+        discountLabel="Discount"
+        discountValue={discount > 0 ? `-${formattedDiscount}` : null}
```
Comment (filipewl) [inline src/components/cart/OrderSummary/OrderSummary.tsx:30]: There is a [compile failure](https://github.com/vtex-sites/nextjs.store/runs/8258301955#:~:text=components/cart/OrderSummary/-,OrderSummary,-.tsx%3A30%3A9) here:
CleanShot 2022-09-09 at 11 59 42@2x
```suggestion
        discountValue={discount > 0 ? `-${formattedDiscount}` : undefined}
```

**Review by filipewl — APPROVED** [review]
Good job on your first "component placement" task! 👏 

I left 2 suggestions for you to take a look.

File: src/components/cart/OrderSummary/order-summary.module.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.fs-order-summary {
+  padding: var(--fs-spacing-3);
+
+  [data-fs-order-summary]li {
```
[the remark this answers is not in the document]
Comment (icazevedo) [reply src/components/cart/OrderSummary/order-summary.module.scss:4]: Great!

File: src/components/cart/OrderSummary/OrderSummary.tsx (hunk 1 above)
Comment (icazevedo) [reply src/components/cart/OrderSummary/OrderSummary.tsx:30]: 👌

**Pull request merged**